src/utils/openSea.ts

```diff
@@ -0,0 +1,36 @@
+import axios from "axios"
+import queryString from "query-string"
+
+import env from "@/.env.json"
+
+const OPENSEA_API_BASE_URL =
+  env.env === "development"
+    ? "https://rinkeby-api.opensea.io/api/v1"
+    : "https://api.opensea.io/api/v1"
+
+type OpenSeaAsset = {
+  token_id: string
+}
+
+export const retrieveOwnerNFTs = async ({ owner }: { owner: string }) => {
+  const contractAddress = env.contractAddress
+
+  const qs = queryString.stringify({
```

Comment:
better use the builtin `URLSearchParams` ?  no need of this dependency

https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

```js
// Search parameters can also be an object
let paramsObj = {foo: 'bar', baz: 'bar'};
let searchParams = new URLSearchParams(paramsObj);

searchParams.toString();                 // "foo=bar&baz=bar"
```